src/parser/index.ts

```diff
@@ -166,12 +171,21 @@ export function parseForESLint(
     );
   }
 
+  const styleElement = ast.body.find(
+    (b): b is SvelteStyleElement => b.type === "SvelteStyleElement"
+  );
+  const styleSourceAst: Root | null =
```

Comment:
I am thinking that maybe it would be useful to have a distinction between 2 states, that currently both produce `null` as a result:
1. There is no `SvelteStyleElement`
2. There is one, but the AST couldn't be parsed

@ota-meshi what do you think? I am not sure what is the right approach here...

Reply:
I think it makes more sense to return more information together. I also think it useful to return information whether it was parsed as CSS or SCSS.

Reply:
That's a great idea, thanks!